### ISSUE TITLE
Nuget should include documentation

### ISSUE THREAD

**Issue by dzmitry-lahoda** [EamonNerbonne/ExpressionToCode#78]
Add XML comments along with dll.

**Comment by EamonNerbonne** [comment]
I'll enable the xml comments I already have embedded anyhow; however I also have a bunch of overloads and places where public stuff is perhaps not fully documented.

Do you have experience with using these doccomments in nuget packages?

Any pointers, PRs, suggestions, or places where docs are particularly relevant would be much appreciated.

**Comment by EamonNerbonne** [comment]
Pushed as 2.6.0

**Comment by dzmitry-lahoda** [comment]
You may look into `FastExpressionCompiler`. When I nuget it I get xml along side with dll. This XML is consumed by VS IDE and I see comments during typing. Just include that XML into nuget, most nugets do it. Thanks.

**Comment by EamonNerbonne** [comment]
Yeah, I'm using `<GenerateDocumentationFile>true</GenerateDocumentationFile>` too; I just don't really feel like copy-pasting the same comment 10 times across all kinds of overloads, and then maintaining all that.

I'll see.
